server/irmaserver/api.go

```diff
@@ -168,8 +168,15 @@ func (s *Server) StartSession(req interface{}, handler server.SessionHandler) (*
 	if err := s.validateRequest(request); err != nil {
 		return nil, "", err
 	}
-
 	if action == irma.ActionIssuing {
+
```

Comment:
Looks like you accidentally placed an empty line here, would prefer it to be removed.